.github/workflows/cockroachdb.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+# https://docs.github.com/en/actions/use-cases-and-examples/building-and-testing/building-and-testing-nodejs
+# https://docs.github.com/en/actions/use-cases-and-examples/using-containerized-services/creating-postgresql-service-containers
+
+name: cockroachdb
+
+on:
```

Comment:
Instead of triggering on each file, I would create one main file that triggers off of this and calls each of the subfiles in parallel.

This will let there be one activity in the GHA view with many steps.

Reply:
I've made is specifically for others to see from the activity what tests passed and what failed (as it's currently made for CircleCI)

Reply:
If we do a single entry point, Github will show each step broken out on the PR like you want.

And in the Activity tab at the top it will be much more clean, since it will group a test/lint run as one item (which you can click on and open up to see each job's details) for that specific PR (instead of several jobs for 1 PR, which makes it difficult to see a single run's results in one place)

Here's an example from a single entry point with multiple steps. And the test results one actually runs 8 parallel test jobs. When the jobs run and complete will show a line item for each with clear messages about pass/fail.

<img width="857" alt="Screenshot 2024-12-06 at 10 30 40 PM" src="https://github.com/user-attachments/assets/69a94f07-fbc0-4afc-9637-6a670ce06bb6">

Reply:
Should I/we do this now or will we leave this for the future?

Reply:
Made it into one file `test.yml`